tools/donate-cpu-server.py

```diff
@@ -518,11 +521,14 @@ def diffMessageIdTodayReport(resultPath: str, messageId: str) -> str:
         if not os.path.isfile(filename):
             continue
         with open(filename, 'rt') as f:
-            diff_stats = f.read()
-        if messageId not in diff_stats:
+            diff_stats = json.loads(f.read())
+        if messageId not in diff_stats['sums']:
+            continue
+        if OLD_VERSION not in diff_stats['sums'][messageId]:
             continue
         if today not in diff_stats:
```

Comment:
This breaks the report and it is no longer a string but an object. Will fix.

Reply:
I have fixed this but cannot test it right now because because requests to daca somehow do not work right now (I have no idea why - restarting it didn't help) and I cannot find a message ID to test it with.

Reply:
Fixed.